src/index.js

```diff
@@ -229,6 +229,11 @@ const initiateUpload = (element, signingUrl, uploadParameters, file, dest) => {
   });
 };
 
+const setFormName = name => {
+  const name_input = document.querySelector('#videoUploadForm input[name=name]');
+  name_input.value = name;
```

Comment:
can you please give me more context on this line?

my concern is if selector does not match any element this line will raise a TypeError.

But I'm not sure if that's a valid assumption, otherwise we need to just check name_input is not null

Reply:
that is possible. I will add that. In the context we will be using this in, it shouldn't happen but it's better to be safe than sorry I suppose :)

Reply:
@semanticmx done